src/main/java/evm/RepositoryMock.java

```diff
@@ -0,0 +1,173 @@
+package evm;
```

Comment:
⚠️  this entire class is not supported by any unit test. What I would expect to see is a unit test for the happy path and unhappy path of every single method. Their _correct functionality_ is very critical. Please add unit testing. Thanks.

Reply:
@KedarKshatriya we cannot merge a class that is not supported by unit testing.

Reply:
Sorry I missed it last time. Fixed here c07d2033efda9ef283c944cc4e13dc20544fb42b